README.md

```diff
@@ -229,15 +238,14 @@ A collection of awesome penetration testing resources
 * [Linux Exploit Suggester](https://github.com/PenturaLabs/Linux_Exploit_Suggester) - Linux Exploit Suggester; based on operating system release number.
 
 #### DDoS Tools
-* [LOIC](https://github.com/NewEraCracker/LOIC/) - An open source network stress tool for Windows
-* [JS LOIC](http://metacortexsecurity.com/tools/anon/LOIC/LOICv1.html) - JavaScript in-browser version of LOIC
```

Comment:
Should be replaced with HOIC if DoS is to remain, LOIC doesn't have anything in place to protect the user, IIRC.